src/main/java/com/backend/controller/PortfolioController.java

```diff
@@ -103,6 +107,37 @@ public GetPortfolioByIdResponse getPortfolio(@PathVariable long pid) {
 		return response;
 	}
 
+	@PutMapping(path = "/portfolio/{pid}")
+	public UpdatePortfolioMetadata updatePortfolio(@PathVariable long pid, @RequestBody Map<String, String> body) {
```

Comment:
should probably replace `Map<String, String>` with `UpdatePortfolioMetadataRequest`

under `./src/main/java/com/backend/request`:

```java
package com.backend.request;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = false)
class UpdatePortfolioMetadataRequest extends BaseRequest {
    String portfolioName;
    String description
}
```

Reply:
resolved 😸  [870608e](https://github.com/is442oop/portfolio-analyzer-backend/pull/35/commits/870608ee218040d9255553e09c3974c0e18ed40f)